src/Sulu/Component/Webspace/Analyzer/Attributes/PortalInformationRequestProcessor.php

```diff
@@ -55,6 +55,10 @@ public function process(Request $request, RequestAttributes $requestAttributes)
         }
 
         $attributes['localization'] = $portalInformation->getLocalization();
+        if ($portalInformation->getLocalization()) {
+            $attributes['locale'] = $portalInformation->getLocalization()->getLocale();
```

Comment:
`$attributes['localization']->getLocale()`

And I think it's not that cool to have a `locale` and a `localization` attribute here, but I guess it is necessary to keep BC, right?